apps/nestor-public/src/pages/view-dashboard.vue

```diff
@@ -1,107 +1,91 @@
 <template>
-  <Page>
-    <PageHeader
-      title="NESTOR Registry"
-      subtitle="Dashboard"
-      titlePositionX="center"
-      titlePositionY="center"
-      height="medium"
-    />
+  <Page id="page-dashboard">
     <LoadingScreen v-if="loading" />
     <div class="page-section padding-h-2" v-else-if="!loading && error">
       <MessageBox type="error">
         <p>Unable to retrieve data {{ error }}</p>
       </MessageBox>
     </div>
     <Dashboard
-      v-else-if="!loading && !error"
-      id="public-dashboard"
+      id="genturisPublicDashboard"
       :verticalPadding="0"
       :horizontalPadding="2"
+      v-else
     >
+      <DashboardRow id="registryHighlights" :columns="1">
+        <DataValueHighlights
+          title="ern genturis registry at a glance"
+          :data="registryHighlights"
+        />
+      </DashboardRow>
       <DashboardRow :columns="2">
         <DashboardChart>
           <GeoMercator
-            :chartId="OrganisationsChart?.chartId"
-            :title="OrganisationsChart?.chartTitle"
-            :chartData="OrganisationsChartData"
+            chartId="registryInstitutionsMap"
+            title="Status of data by healthcare provider"
+            :chartData="organisations"
             rowId="code"
             longitude="longitude"
             latitude="latitude"
+            :geojson="WorldGeoJson"
             group="hasSubmittedData"
-            :legendData="OrganisationsChartPalette"
-            :groupColorMappings="OrganisationsChartPalette"
-            :geojson="geojson"
+            :groupColorMappings="orgGroupMapping"
+            :legendData="orgGroupMapping"
             :chartSize="114"
-            :chartHeight="350"
+            :chartHeight="190"
             :mapCenter="{
-              // centroid of the Netherlands
-              latitude: 5.291266,
-              longitude: 52.132633,
+              latitude: 5,
+              longitude: 51,
             }"
-            :pointRadius="6"
-            :tooltipTemplate="(row: IOrganisations) => {
-              return `
+            :pointRadius="4"
+            :tooltipTemplate="
+              (row: IOrganisations) => {
+                return `
                 <p class='title'>${row.name}</p>
                 <p class='location'>${row.city}, ${row.country}</p>
-                `;
-            }"
-            :mapColors="{
-              water: 'hsl(177, 63%, 90%)',
-              land: 'hsl(177, 63%, 35%)',
-              border: 'hsl(177, 63%, 90%)',
-            }"
-            :chartScale="10"
-            :zoomLimits="[0, 10]"
+            `;
+              }
+            "
+            :zoomLimits="[0.3, 10]"
```

Comment:
Since the map is specific to the Netherlands, perhaps zooming in a bit more would be helpful. We could also look into creating new base layer of just the Netherlands if that's better.